python/cudf/cudf/io/parquet.py

```diff
@@ -193,16 +194,16 @@ def write_to_dataset(
 
 @ioutils.doc_read_parquet_metadata()
 @_cudf_nvtx_annotate
-def read_parquet_metadata(path):
-    """{docstring}"""
-
-    pq_file = pq.ParquetFile(path)
-
-    num_rows = pq_file.metadata.num_rows
-    num_row_groups = pq_file.num_row_groups
-    col_names = pq_file.schema.names
-
-    return num_rows, num_row_groups, col_names
+def read_parquet_metadata(path, fs=None, storage_options=None):
+    fs = (
+        fs
+        or fsspec.core.get_fs_token_paths(
+            path, storage_options=storage_options or {}
+        )[0]
+    )
```

Comment:
This is a bit hard to read. Maybe better to use an explicit `if` statement here
```suggestion
    if fs is None:
        fs = fsspec.core.get_fs_token_paths(
            path, storage_options=storage_options or {}
        )[0]
```